ApiDemos/kotlin/app/src/main/java/com/example/kotlindemos/CameraDemoActivity.kt

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.kotlindemos
+
+import android.graphics.Color
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import android.view.View
+import android.widget.CompoundButton
+import android.widget.SeekBar
+import android.widget.Toast
+import com.google.android.gms.maps.CameraUpdate
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.GoogleMap.CancelableCallback
+import com.google.android.gms.maps.GoogleMap.OnCameraIdleListener
+import com.google.android.gms.maps.GoogleMap.OnCameraMoveCanceledListener
+import com.google.android.gms.maps.GoogleMap.OnCameraMoveListener
+import com.google.android.gms.maps.GoogleMap.OnCameraMoveStartedListener
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.PolylineOptions
+
+/**
+ * This shows how to change the camera position for the map.
+ */
+class CameraDemoActivity :
+        AppCompatActivity(),
+        OnCameraMoveStartedListener,
+        OnCameraMoveListener,
+        OnCameraMoveCanceledListener,
+        OnCameraIdleListener,
+        OnMapReadyCallback {
+
+    /**
+     * The amount by which to scroll the camera. Note that this amount is in raw pixels, not dp
+     * (density-independent pixels).
+     */
+    private val SCROLL_BY_PX = 100
+    private val TAG = CameraDemoActivity::class.java.name
+    private val sydneyLatLng = LatLng(-33.87365, 151.20689)
+    private val bondiLocation: CameraPosition = CameraPosition.Builder().
+            target(LatLng(-33.891614, 151.276417))
```

Comment:
Should the dot be in the same line as target?

Reply:
Done!